llvm/lib/Transforms/Utils/InjectTLIMappings.cpp

```diff
@@ -35,35 +35,36 @@ STATISTIC(NumCompUsedAdded,
 
 /// A helper function that adds the vector function declaration that
 /// vectorizes the CallInst CI with a vectorization factor of VF
-/// lanes. The TLI assumes that all parameters and the return type of
-/// CI (other than void) need to be widened to a VectorType of VF
-/// lanes.
+/// lanes. For each mapping TLI contains a VABI prefix, which contains
```

Comment:
nit: I would rewrite maybe to something like:
For each mapping, TLI provides/has a VABI prefix, which contains ...

Reply:
Done